libred/red.c

```diff
@@ -44,6 +44,18 @@
 #include <linux/unistd.h>
 #include <sys/time.h>
 
+#define str(s) #s
+#define xstr(s) str(s)
+
+/* Redirection of tool invocations that match absolute prefixes */
+static char const *prefixes[] = {
+  "/usr/bin",
+  "/bin",
+  "/usr/sbin"
+};
+#define N_PREFIXES (sizeof(prefixes) / sizeof(prefixes[0]))
```

Comment:
I'd add pass the array as an argument and rename the macro to `array_size`, that way it can be used with arbitrary array not just this one.